testing/tests/ui/let_destructuring_has_rest.rs

```diff
@@ -0,0 +1,40 @@
+use rinja::Template;
+
+struct X {
+    a: u32,
+    b: u32,
+}
+
+#[derive(Template)]
+#[template(source = "
+{%- if let X { a, .. } = x -%}hello {{ a }}{%- endif -%}
```

Comment:
Same: this one is valid so I think it'd be better to move it into non-ui tests.

Reply:
Yep, also moved.